configuration.go

```diff
@@ -119,10 +132,22 @@ func (config *Configuration) loadEnv() {
 		envConfig.APIKey = apiKey
 	}
 
-	if appVersion := os.Getenv("BUGSNAG_APP_VERSION"); appVersion != "" {
+	if serviceName := os.Getenv("BUGSNAG_PERFORMANCE_SERVICE_NAME"); serviceName != "" {
+		envConfig.ServiceName = serviceName
+	} else if serviceName := os.Getenv("OTEL_SERVICE_NAME"); serviceName != "" {
+		envConfig.ServiceName = serviceName
+	}
```

Comment:
As per comment above - do we need to do this or will it already be in the default resources from the OTel SDK?

Reply:
added tests - it's handled by default resource values